clambcc/clambc-compiler.py

```diff
@@ -444,16 +468,27 @@ def getOutputString(linked: IRFile, ignore: IRFile) -> str:
 def createOptimizedTmpFile(clangLLVM: ClangLLVM, linkedFile: str) -> str:
     name = getOptimizedTmpFileName(linkedFile)
 
-    cmd = f"{clangLLVM.getOpt()} \
+    cmd = f'{clangLLVM.getOpt()} \
             -S \
             {linkedFile} \
             -o {name} \
-            -internalize -internalize-public-api-list=entrypoint \
-            -globalopt \
-            "
+            -internalize-public-api-list=entrypoint \
+            --passes="internalize,globalopt" \
+            '
+
+    cmd = []
+    cmd.append(clangLLVM.getOpt())
+    cmd.append("-S")
+    cmd.append(linkedFile)
+    cmd.append("-o")
+    cmd.append(name)
+    cmd.append("-internalize-public-api-list=entrypoint")
+    cmd.append('--passes="internalize,globalopt"')
 
     ret = run(cmd)
     if None == ret:
+        print ("remoev me")
+        import pdb ; pdb.set_trace()
```

Comment:
Do you want to replace this with some error message?

Reply:
The message will be provided by opt, but I do want to remove the breakpoint.